sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -491,12 +503,13 @@ def test_series_tests(self):
                 # Inspection after modification.
                 's'
             ],
+            'pandas.core.series.Series.resample': ['df'],
         })
     self.assertEqual(result.failed, 0)
 
   def test_string_tests(self):
-    PD_VERSION = tuple(int(v) for v in pd.__version__.split('.'))
-    if PD_VERSION < (1, 2, 0):
+    PD_VERSION = tuple(int(v) for v in pd.__version__.split('.')[:2])
```

Comment:
Related to `PD_VERSION`:
- Should we define this in one common place?
- And use a consistent way to get it, either tuple(int(... or tuple(map(... for consistency?
- And is it worth keeping the patch version or not? (probably not.)

Reply:
Rewritten to get the PD_VERSION from frames.py. Probably not worth it to keep the patch veresion.